pgcli/pgcompleter.py

```diff
@@ -34,8 +35,6 @@
 
 
 Match = namedtuple('Match', ['completion', 'priority'])
-
-
 class PGCompleter(Completer):
```

Comment:
Can you put back the whitespace before the class definition? It looks really weird without it